AWSAuthSDK/Sources/AWSAuthCore/AWSAppleSignIn/AWSAppleSignInProvider.m

```diff
@@ -86,16 +86,13 @@ - (void)reloadSession {
 #pragma mark -
 
 - (void)appleLogin API_AVAILABLE(ios(13)) {
-    if (@available(iOS 13.0, *)) {
-
-        ASAuthorizationAppleIDProvider *appleIDProvider = [ASAuthorizationAppleIDProvider new];
-        ASAuthorizationAppleIDRequest *request = appleIDProvider.createRequest;
-        request.requestedScopes = @[ASAuthorizationScopeFullName, ASAuthorizationScopeEmail];
-        ASAuthorizationController *controller = [[ASAuthorizationController alloc] initWithAuthorizationRequests:@[request]];
-        controller.delegate = self;
-        controller.presentationContextProvider = self;
-        [controller performRequests];
-    }
+    ASAuthorizationAppleIDProvider *appleIDProvider = [ASAuthorizationAppleIDProvider new];
+    ASAuthorizationAppleIDRequest *request = appleIDProvider.createRequest;
+    request.requestedScopes = @[ASAuthorizationScopeFullName, ASAuthorizationScopeEmail];
```

Comment:
Is hardcoding this the right way to go about this? I know that HostedUI allows for passing custom scopes, as well as defining a list of scopes in the config. What about drop in UI?

Reply:
Thanks for catching this. Change the code to follow the same approach as in other social provider.